stopify/src/runtime/abstractRunner.ts

```diff
@@ -28,6 +41,12 @@ export abstract class AbstractRunner implements AsyncRun {
   private eventQueue: EventHandler[] = [];
   private higherOrderFunctions: any;
 
+  private onYieldFlag: onYieldFlagValues = onYieldFlagValues.Resume;
+  private mayYieldFlag: mayYieldFlagValues = mayYieldFlagValues.Resume;
+  private captureCurrentLine: number | undefined;
+  private capturePauseFn: (line?: number) => void;
+  private captureOnStepFn: (line: number) => void;
```

Comment:
The two method signatures (copied from below) differ only in if `line` is required. Shouldn't it be optional in both cases (based on our discussion of line number support), or am I missing something?

Also,  I was inconsistent in naming conventions; will update in next commit.